.azure-pipelines/docker.yml

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the ODPi Egeria project.
+---
+trigger:
+  paths:
+    include:
+      - open-metadata-resources/open-metadata-deployment/docker
+pr: none
+schedules:
+  - cron: "11 0 * * *"
+    displayName: Daily Docker Build
+    always: true
+    branches:
+      include:
+        - master
+
+stages:
+  - stage: Docker
+    jobs:
+      - job: Maven
+        pool:
+          vmImage: 'ubuntu-latest'
+        strategy:
+          matrix:
+            apache-atlas:
+              POM_FILE: 'open-metadata-resources/open-metadata-deployment/docker/apache-atlas/pom.xml'
+            egeria:
+              POM_FILE: 'open-metadata-resources/open-metadata-deployment/docker/egeria/pom.xml'
```

Comment:
The current egeria docker build is broken. I'm fixing currently, but it's strategy is to make use of our maven artifacts. currently the Dockerfile refers to the odpi registry as the default - we'll need it to point to wherever the right url is for snapshots in jfrog. The repo is a parameter to the dockerfile, but I've not embraced that in the pom yet... I'll raise a seperate issue to correct this after I've fixed the egeria docker build. (The reason for the change was to avoid re-extracting from git and running a 40-60 min process when we could just use maven?)

Reply:
Right there's no need to rebuild the world when all you want to do is package things into a Docker container. It sounds like what you'd like is to have these builds trigger when the new snapshot (and release) jars are created, so that they're hopefully always up to date with the current builds and it's easy for users to pull down and run them. I'm pretty sure that can be done through AZP. It's probably best to meet and write down these future improvements so we're all clear on how the containers are setup and relate to the code/artifacts, as I'm still trying to figure the big picture. :)

Reply:
ok thanks for the info. will close the thread here and open up a new discussion - probably once we have the basics in place